code/game/objects/items/weapons/storage/belt.dm

```diff
@@ -488,3 +489,36 @@
 		"/obj/item/weapon/mop",
 		"/obj/item/weapon/storage/bag/trash")
 
+/obj/item/weapon/storage/belt/leather
+	name = "leather belt"
+	desc = "This belt is of uncommonly good craftsmanship. For some reason, the inside of the buckle reads '18+'."
+	icon_state = "leather"
+	item_state = "leather"
+	storage_slots = 2 //no pouches, no straps, just a waistband
+
+/obj/item/weapon/storage/belt/leather/equipped(mob/living/M, slot)
+	..()
```

Comment:
Check if there is some return value on equipped, just to be sure you can't just spam it on an incompatible slot to call equipped() a bunch

Reply:
Tested, doesn't work, return value is not used & the slot is checked just below